completions/kldload

```diff
@@ -7,14 +7,26 @@ _kldload()
     local cur prev words cword
     _init_completion || return
 
-    local moddir=/modules/
-    [[ -d $moddir ]] || moddir=/boot/kernel/
+    if [[ "$cur" == */* ]]; then
```

Comment:
We have some more elaborate checks for what looks like a path in the tree, will convert this to use it after the merge.